### DISCUSSION TITLE
I'm struggling with replication from Harbor to Azure ACR

### DISCUSSION THREAD

**Discussion by PJSFL** [goharbor/harbor#18333] [Q&A]
Hello,

Guys, I need help with Harbor to Azure replication.
I've been struggling with the problem of replication not executing for two days now, and I think it's time to give up and seek advice here.

I set up a repository on our new Harbor server, I configured replication and when I run the replication the task ends Failed or Successful but still the repository in Azure does not receive any files.
I'd like to have Azure as a backup for Harbor and a potential place to download application images if the customer prefers Azure to Harbor.

![image](https://user-images.githubusercontent.com/127487190/224276311-0bb6636b-072d-4ec9-90d2-407153424778.png)

here everything looks fine, but sometimes i recieve a message 'Failed to connect to endpoint' or the status 'unhealthy' - i don't know why.

![image](https://user-images.githubusercontent.com/127487190/224276805-7cef8f54-72b5-467e-bd3d-d5382f56196e.png)

and when i run the replication, all of the files are failed

![image](https://user-images.githubusercontent.com/127487190/224277027-a9d3a24c-dc08-48a8-8bd8-71f7af28c02e.png)

I can see some errors in logs but have no idea how to fix them

a) failed to push manifest of artifact

b) 
2023-03-10T09:09:05Z [ERROR] [/controller/replication/transfer/image/transfer.go:481]: failed to check the existence of the manifest of artifact XXX/mariadb:latest on the destination registry: http status code: 403, body: 
2023-03-10T09:09:05Z [ERROR] [/controller/replication/transfer/image/transfer.go:194]: http status code: 403, body: 
2023-03-10T09:09:05Z [ERROR] [/controller/replication/transfer/image/transfer.go:200]: got error during the whole transfer period, mark the job failure
: the blob sha256:XXX already exists on the destination registry, skip

but there is not a single file there on Azure ACR...

If you had any ideas - I would appreciate your support.
Azure ACR has a premium version. In Access ID and Password, I entered the login and password from the Access Keys tab in the Azure container. I didn't configure anything beyond that.

Thanks!

**Comment by PJSFL** [comment]
Bump

**Comment by wy65701436** [comment]
can you help to transfer it into a github issue?

**Comment by PJSFL** [comment]
@wy65701436 How can I transfer it?
